features/extensions/azure/install.py

```diff
@@ -96,12 +96,13 @@ def install(jazz_stackprefix, jenkins_url, jenkins_user, jenkins_api_token, jazz
     startJob(jenkins_url, jenkins_user, jenkins_api_token, "job/jazz_ui/buildWithParameters?token=jazz-101-job")
 
 
-def prepare_azure_json(azure_location, azure_subscription_id):
+def prepare_azure_json(azure_location):
     extension_base = "extensions/azure"
+    azure_stat_accnt = 'AZURE_POC_DEV_TEST'
     azureConfig = OrderedDict()
     azureConfig['IS_ENABLED'] = True
     azureAccount = OrderedDict()
-    azureAccount['ACCOUNTID'] = azure_subscription_id
+    azureAccount['ACCOUNTID'] = azure_stat_accnt
     azureAccount['ACCOUNTNAME'] = 'azureaccount'
     azureAccount['SUBSCRIPTION_ID'] = 'AZ_SUBSCRIPTIONID'
```

Comment:
I understand that we are storing this in secret store but subscriptionId can live in admin configurations directly. You can keep this as-is for now.